### DISCUSSION TITLE
HTTPX Provides Screenshot

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1046] [General]
I noticed httpx added ability to get screenshot, maybe helpful.

![image](https://github.com/blacklanternsecurity/bbot/assets/15929497/fb8bc2c3-683c-4c0b-9d6c-9d398a922840)

**Comment by TheTechromancer** [comment]
Yes I had noticed this. It's possible it might end up being an alternative to gowitness, but Gowitness has some very nice features such as automatic tech detect and a web server to browse. To be honest I'm a little annoyed they chose to add this feature, since it slows down httpx significantly even when it isn't enabled (it takes a full two seconds longer to start the program, which may not sound like a lot, but can really add up). 

Until we get around to evaluating this, we've pinned the older, faster version of httpx.
